templates/resources/common/bin/bootstrapper.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
```

Comment:
How does the `bootstrapper.sh` relate to the `run-in-container.sh`? This one does not seem very generic because it mentions `xlr` all over the place.

Reply:
Shite, shouldn't have been committed :D